examples/05_glm_second_level/plot_second_level_one_sample_test.py

```diff
@@ -94,47 +115,94 @@
-                             smoothing_fwhm=8.0, n_jobs=1)
-
-###########################################################################
-# Let us plot the (corrected) negative log p-values for the nonparametric test.
-title = ('Group left-right button press: \n'
-         'permutation test (FWER < 10%)')
-display = plotting.plot_glass_brain(
-    neg_log_pvals_permuted_ols_unmasked, colorbar=True, vmax=3,
-    display_mode='z', plot_abs=False, cut_coords=cut_coords,
-    threshold=threshold, title=title)
-plotting.show()
 
-# The neg-log p-values obtained with nonparametric testing are capped at 3
-# since the number of permutations is 1e3.
+out_dict = non_parametric_inference(
+    second_level_input,
+    design_matrix=design_matrix,
+    model_intercept=True,
+    n_perm=500,  # 500 for the sake of time. Ideally, this should be 10,000.
+    two_sided_test=False,
+    smoothing_fwhm=8.0,
+    n_jobs=1,
+    threshold=0.001,
+)
+
+###############################################################################
+# Let us plot the (corrected) negative log p-values for the both tests.
+#
+# We will use a negative log10 p threshold of 1, which corresponds to p<0.1.
+# This threshold indicates that there is less than 10% probability to make a
+# single false discovery (90% chance that we make no false discovery at all).
+# This threshold is much more conservative than an uncorrected threshold, but
+# is still more liberal than a typical corrected threshold for this kind of
+# analysis, which tends to be ~0.05.
+#
+# We will also cap the negative log10 p-values at 2.69, because this is the
+# maximum observable value for the nonparametric tests, which were run with
+# only 500 permutations.
+threshold = 1  # p < 0.1
+vmax = 2.69  # ~= -np.log10(1 / 500)
+
+cut_coords = [0]
+
+IMAGES = [
+    neg_log_pval,
+    out_dict['logp_max_t'],
+    out_dict['logp_max_size'],
+    out_dict['logp_max_mass'],
+]
+TITLES = [
+    'Parametric Test',
+    'Permutation Test\n(Voxel-Level Error Control)',
+    'Permutation Test\n(Cluster-Size Error Control)',
+    'Permutation Test\n(Cluster-Mass Error Control)',
+]
+
+fig, axes = plt.subplots(figsize=(8, 8), nrows=2, ncols=2)
+img_counter = 0
+for i_row in range(2):
+    for j_col in range(2):
+        ax = axes[i_row, j_col]
+        plotting.plot_glass_brain(
+            IMAGES[img_counter],
+            colorbar=True,
+            vmax=vmax,
+            display_mode='z',
+            plot_abs=False,
+            cut_coords=cut_coords,
+            threshold=threshold,
+            figure=fig,
+            axes=ax,
+        )
+        ax.set_title(TITLES[img_counter])
+        img_counter += 1
+
+fig.suptitle('Group left-right button press\n(negative log10 p-values)')
+
 # The nonparametric test yields many more discoveries and is more powerful than
 # the usual parametric procedure.
+# Even within the nonparametric test, the different correction metrics produce
```

Comment:
These lines render as a comment in the code cell. Maybe we could swap them and the `plt.show()` line above, and make them proper text?

Reply:
Great idea! I've changed it in [f31927e](https://github.com/nilearn/nilearn/pull/3181/commits/f31927e116640f2990301af3f20fb169ea17dce6).